src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js

```diff
@@ -0,0 +1,130 @@
+import React, { useCallback } from "react";
+import { useKeyEvent } from "hooks";
+import Clickable from "../Clickable/Clickable";
+import Flex from "../Flex/Flex";
+import Avatar from "../Avatar/Avatar";
+import avatarGroupCounterTooltipContentStyles from "./AvatarGroupCounterTooltipContent.module.scss";
+
+const KEYS = ["Tab"];
+
+export function useTooltipContentTabNavigation({
+  counterContainerRef = undefined,
+  tooltipContentContainerRef,
+  focusPrevPlaceholderRef,
+  focusNextPlaceholderRef,
+  setShouldUpdate,
+  setIsTooltipVisible
+}) {
+  const hideTooltip = useCallback(() => {
+    // Tricky way to close the tooltip
+    setTimeout(() => {
+      setIsTooltipVisible(false);
+      setIsTooltipVisible(true);
+    });
+  }, [setIsTooltipVisible]);
+
+  // For Counter
+  useKeyEvent({
+    keys: KEYS,
+    withoutAnyModifier: true,
+    ref: counterContainerRef,
+    callback: useCallback(
+      e => {
+        if (e.target === counterContainerRef.current) {
+          e.preventDefault();
+          tooltipContentContainerRef.current && tooltipContentContainerRef.current.focus();
+          setShouldUpdate(prev => !prev);
+        }
+      },
+      [counterContainerRef.current, setShouldUpdate, tooltipContentContainerRef]
```

Comment:
see the lint comment

Reply:
I've seen the tip, but it does work with `ref.current` and doesn't work with just `ref`
Not sure how to make it work some other way. Maybe some explanation would help